src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -820,6 +841,9 @@ export class PreferenceStore {
             PreferenceKeys.GLOBAL_AUTOLAUNCH,
             PreferenceKeys.GLOBAL_FILE_FILTER_MODE,
             PreferenceKeys.GLOBAL_LAYOUT,
+            PreferenceKeys.GLOBAL_DYNAMIC_LAYOUT_ENABLE,
+            PreferenceKeys.GLOBAL_DYNAMIC_LAYOUT,
```

Comment:
This removes all layout connections when clicking restore default in the global tab. Perhaps we need a new function for the Layout tab.